Apps/WebClient/src/ClientApp/src/components/communication.vue

```diff
@@ -1,9 +1,17 @@
+<style lang="scss" scoped>
+@import "@/assets/scss/_variables.scss";
+
+.communication {
+  background-color: $bcgold;
+  color: black;
+}
+</style>
 <template>
   <b-row v-if="communication">
```

Comment:
looking below the line looks kinda odd
```
<b-row v-if="communication">
...
  private communication: Communication | null = null;
```
How about instead using a calculated member
```
 private communication?: Communication;
private get hasComunication():boolean {
   return this.communication !== undefined;
}
```